lib/objectBuilder.js

```diff
@@ -104,7 +104,8 @@ module.exports = function () {
     var rec = _.assign(_createObject(), deviceInfo, {
       type: 'smbg',
       value: REQUIRED,
-      units: REQUIRED
+      units: REQUIRED,
+      annotation: OPTIONAL
```

Comment:
Hmm, in the other drivers I've abstracted out some annotation functions instead of adding them as optional in the builder. This makes quite a bit of sense because the annotateEvent function needs to preserve existing annotations and check for duplicates before adding a new one. (This kind of thing can happen in basals.)

But we could just merge this and update later, since the common annotations functions are all the way over on my Insulet branch right now?